lib/objects/objectsInMemServerRedis.js

```diff
@@ -590,6 +591,28 @@ class ObjectsInMemoryServer extends ObjectsInMemoryFileDB {
             }
         });
 
+        handler.on('exists', async (data, responseId) => {
```

Comment:
really only support one key? ;-) Redis allows also multiple ... https://redis.io/commands/exists we should minimum add a comment

Reply:
Hm, for the current use case it is enough to support one key.. we can add a comment or extend the functionality.

Reply:
Hm after checking I would only add a comment. 
Because it most likely be that we will never need the exists check for more than one key, because `EXISTS` is always an 'or' check when providing multiple keys - I thought I might get an array for every key which could have been a use case due to increased performance.

Reply:
Hm.. okay, redis then would return total number of existing keys, so we could check easily if all given files exist.. so it may be useful..